README.md

```diff
@@ -97,22 +104,29 @@ create virtual machines for hosting the containers.
 
 ##### Kernel
 
-The [`linux`](https://github.com/kata-containers/linux) kernel is used by the hypervisor to boot the guest image.
+The [Linux\*](https://github.com/kata-containers/linux) kernel is used by the hypervisor to boot the guest image.
```

Comment:
Passive voice. Consider:
The hypervisor uses the  [Linux\* kernel](https://github.com/kata-containers/linux) to boot the guest image.